terraform/modules/quorum-cluster-region/user-data/user-data-bootnode.sh

```diff
@@ -43,6 +43,26 @@ function run_threatstack_agent_if_key_provided {
   fi
 }
 
+function setup_s3fs {
+  echo "${constellation_s3_bucket} /opt/quorum/constellation/private/s3fs fuse.s3fs _netdev,allow_other,iam_role 0 0" | sudo tee /etc/fstab
+  sudo mount -a
+}
+
+function populate_data_files {
+  echo "${index}" | sudo tee /opt/quorum/info/index.txt
+  echo "${bootnode-count-json}" | sudo tee /opt/quorum/info/bootnode-counts.json
+  sudo python /opt/quorum/bin/fill-node-counts.py --quorum-info-root '/opt/quorum/info' --bootnode
+  echo "${aws_region}" | sudo tee /opt/quorum/info/aws-region.txt
+  echo "${primary_region}" | sudo tee /opt/quorum/info/primary-region.txt
+
+  #TODO: Init scripts used to use this file to determine when provisioning is done.
```

Comment:
We can just throw that whole mechanism out.

Since everything is run by user-data now, we just have it write the data first and we know the data is there. This was to fix a race condition between the user-data and the provisioner.